devtools-ci-index.yaml

```diff
@@ -97,7 +97,28 @@
             # testing out the cico client
             set +e
             export CICO_API_KEY=$(cat ~/duffy.key )
-            read CICO_hostname CICO_ssid <<< $(cico node get -f value -c ip_address -c comment)
+            # get node
```

Comment:
all good! Just it was the same code repeated again and again. Perhaps we could re-use it as a template. I can do that in my iteration.